Documentation/ceph-pool-radosnamespace.md

```diff
@@ -0,0 +1,44 @@
+---
+title: RadosNamespace CRD
+weight: 3610
+indent: true
+---
+
+{% include_relative branch.liquid %}
+
+This guide assumes you have created a Rook cluster as explained in the main [Quickstart guide](quickstart.md)
+
+# CephBlockPoolRadosNamespace CRD
+
+Rook allows creation of Ceph BlockPool
+[RadosNamespaces](https://docs.ceph.com/en/latest/man/8/rbd/) through the
+custom resource definitions (CRDs). BlockPool Rados Namespace is an abstraction
+inside a block pool. For more information about BlockPool and namespace refer to
+the [Ceph docs](https://docs.ceph.com/en/latest/man/8/rbd/).
```

Comment:
Could you add a sentence that gives an idea why a namespace would be useful or when it should be used?

Reply:
sure will do it.